account/src/main/java/org/killbill/billing/account/dao/DefaultAccountDao.java

```diff
@@ -317,4 +323,15 @@ public Integer inTransaction(final EntitySqlDaoWrapperFactory entitySqlDaoWrappe
             }
         });
     }
+
+    @Override
+    public List<AuditLogWithHistory> getAuditLogsWithHistoryForId(final UUID accountId, final AuditLevel auditLevel, final InternalTenantContext context) throws AccountApiException {
+        return transactionalSqlDao.execute(false, AccountApiException.class, new EntitySqlDaoTransactionWrapper<List<AuditLogWithHistory>>() {
+            @Override
+            public List<AuditLogWithHistory> inTransaction(final EntitySqlDaoWrapperFactory entitySqlDaoWrapperFactory) throws EntityPersistenceException, EventBusException {
```

Comment:
I believe you can simplify the exceptions (`EntityPersistenceException` and `EventBusException` aren't thrown).